scripts/jenkins/ardana/ansible/bootstrap-clm.yml

```diff
@@ -25,26 +25,24 @@
     task: "deploy"
 
   pre_tasks:
-    - block:
-        - name: Grow 1st partition filesystem
-          shell: |
-            /usr/sbin/growpart /dev/vda 1 && /sbin/resize2fs /dev/vda1
-          failed_when: False
-          when: not is_physical_deploy
+     - block:
+         - include_role:
+             name: setup_root_partition
+           when: not is_physical_deploy
 
-        - include_role:
-            name: "setup_zypper_repos"
-      rescue:
-        - include_role:
-            name: rocketchat_notify
-          vars:
-            rc_action: "finished"
-            rc_state: "Failed"
-          when: rc_notify
+         - include_role:
+             name: "setup_zypper_repos"
+       rescue:
+         - include_role:
+             name: rocketchat_notify
+           vars:
+             rc_action: "finished"
+             rc_state: "Failed"
+           when: rc_notify
 
-        - name: Stop if something failed
-          fail:
-            msg: "{{ task }} failed."
+         - name: Stop if something failed
+           fail:
+             msg: "{{ task }} failed."
 
```

Comment:
An inconsistent white space. Here it is 3 spaces and 2 everywhere else.

Reply:
done